backend/curriculum_tracking/serializers.py

```diff
@@ -326,6 +330,13 @@ def get_users_that_reviewed_open_prs(self, instance):
     def get_users_that_reviewed_open_prs_emails(self, instance):
         return [o.email for o in instance.get_users_that_reviewed_open_prs()]
 
+    def get_total_number_of_negative_reviews(self, instance):
+        total_number_of_negative_reviews = len(
+            LogEntry.objects.filter(effected_user=instance.assignees.first(), 
```

Comment:
This query gets the total number of times any card belonging to the user was bounced back.

You need to filter by the actual card we are looking at. 

If I have 2 cards:

card_1 bounced 2 times and card_2 bounced 3 times then the total bounces for card_1 is 2 and not 5.

Reply:
please write a test